transit/migrations/0001_initial.py

```diff
@@ -0,0 +1,366 @@
+                ('funding_source', models.CharField(blank=True, db_column='FundingSource', max_length=255, null=True)),
+                ('item_product', models.ForeignKey(db_column='ItemID', on_delete=django.db.models.deletion.DO_NOTHING, to='transit.item')),
+            ],
+            options={
+                'db_table': 'ItemDetails',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='ModeOfTransport',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('last_modified_date', models.DateTimeField(db_column='LastModifiedDate', default=datetime.datetime.now)),
+                ('last_modified_by', models.CharField(db_column='LastModifiedBy', max_length=255)),
+                ('class_mode', models.CharField(blank=True, db_column='Class', max_length=255, null=True)),
+                ('vehicle_type', models.CharField(blank=True, db_column='VehicleType', max_length=255, null=True)),
+            ],
+            options={
+                'db_table': 'ModeOfTransport',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='OrderDetails',
+            fields=[
+                ('last_modified_date', models.DateTimeField(db_column='LastModifiedDate', default=datetime.datetime.now)),
+                ('last_modified_by', models.CharField(db_column='LastModifiedBy', max_length=255)),
+                ('order_details_id', models.CharField(db_column='OrderDetailsID', max_length=50, primary_key=True, serialize=False, unique=True)),
+                ('order_received_date', models.CharField(blank=True, db_column='OrderReceivedDate', max_length=255, null=True)),
+                ('customer', models.ForeignKey(db_column='CustomerID', on_delete=django.db.models.deletion.DO_NOTHING, to='transit.customer')),
+            ],
+            options={
+                'db_table': 'OrderDetails',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='OrderLineDetails',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('last_modified_date', models.DateTimeField(db_column='LastModifiedDate', default=datetime.datetime.now)),
+                ('last_modified_by', models.CharField(db_column='LastModifiedBy', max_length=255)),
+                ('quantity', models.DecimalField(blank=True, db_column='Quantity', decimal_places=2, max_digits=18, null=True)),
+                ('old_quantity', models.DecimalField(blank=True, db_column='OldQuantity', decimal_places=2, max_digits=18, null=True)),
+                ('item_details', models.ForeignKey(db_column='ItemDetailsId', on_delete=django.db.models.deletion.DO_NOTHING, to='transit.itemdetails')),
```

Comment:
I see that we have some entities with capital `ID` suffix and some of them are `Id`, I'm not sure if that's and issue as sql is case insensitive but we should make them consistent

Reply:
The same as I wrote in one of comments. Unfortunately we have 'inconsistency' in the 'naming' of fields. Sometimes we have 'ID' and sometimes 'Id'. But I tried to keep 1:1 as it is on legacy version for the fields. 

I could make them consistent as you wish, it shouldn't take a lot of time to do this.

Reply:
Thanks for the change, keeping them consistent shouldn't make data migration more complicated so it's good that we fixed it early.